android/BOINC/app/src/main/java/edu/berkeley/boinc/rpc/Message.java

```diff
@@ -64,6 +64,8 @@ private Message(Parcel in) {
         seqno = in.readInt();
         timestamp = in.readLong();
         body = in.readString();
+        body = body.replace("<![CDATA[\n", "");
```

Comment:
Strictly speaking this removes more than the [client change](https://github.com/BOINC/boinc/commit/1fe02c9d4fe815a69096e5cfc8e8985aa2308890#diff-254fd9eea50fb57d93c683e033cd3ad8R220) added, namely the line breaks. Is this intentional?

Reply:
Yes, otherwise before and after every message in event log will be empty lines

Reply:
Ok, but that's how the client is written. I don't think we should fix things locally as this would be inconsistent. If that those extra line breaks aren't wanted then this should be fixed (glabally) in the client.

Reply:
OK, no problem. Removed line breaks removes.